X16 Reference - 04 - BASIC.md

```diff
@@ -501,7 +498,7 @@ The full set of macros is documented [here](X16%20Reference%20-%20Appendix%20A%2
 150 FMCHORD 1,"RRR" : REM RELEASE THE CHANNELS THAT ARE PLAYING THE CHORD
 ```
 
-This will play the first few lines of *Silent Night* with a vibraphone lead and organ accompaniment.
+This will play the first few lines of _Silent Night* with a vibraphone lead and organ accompaniment.
```

Comment:
Change in italics style gone wrong. I don't really see why you've changed all instances of italics from asterisks to underscores, but that's another topic.

Reply:
I have no idea what happened there. The Markdown linter was complaining about the * italics (apparently, the preferred standard is _), and I told it to just fix them. I guess it didn't do a good job. Ugh.